src/backend/tests/change-requests.test.ts

```diff
@@ -0,0 +1,254 @@
+    const invalidWBSElement = { ...redesignWhipWBSElement, workPackage: invalidWP };
+    jest.spyOn(prisma.wBS_Element, 'findUnique').mockResolvedValueOnce(invalidWBSElement);
+    jest.spyOn(prisma.project, 'findUnique').mockResolvedValueOnce(null);
+    const response = await request(app)
+      .post('/review')
+      .send({ ...redesignWhip, psId: '1' });
+    expect(response.status).toBe(404);
+    expect(response.body).toStrictEqual({
+      message: `Work package project not found`
+    });
+    expect(prisma.user.findUnique).toHaveBeenCalledTimes(1);
+    expect(prisma.change_Request.findUnique).toHaveBeenCalledTimes(1);
+    expect(prisma.scope_CR.findUnique).toHaveBeenCalledTimes(1);
+    expect(prisma.proposed_Solution.findUnique).toHaveBeenCalledTimes(1);
+    expect(prisma.wBS_Element.findUnique).toHaveBeenCalledTimes(1);
+    expect(prisma.project.findUnique).toHaveBeenCalledTimes(1);
+  });
+  test('proposedSolutionSuccessfullyImplementedForWorkPackageChange', async () => {
+    jest.spyOn(prisma.user, 'findUnique').mockResolvedValueOnce(superman);
+    jest.spyOn(prisma.change_Request, 'findUnique').mockResolvedValueOnce({ ...redesignWhip, accepted: false });
+    jest.spyOn(prisma.scope_CR, 'findUnique').mockResolvedValueOnce(redesignWhipScopeCR);
+    jest.spyOn(prisma.proposed_Solution, 'findUnique').mockResolvedValueOnce(solutionToRedesignWhip);
+    const updatedSolution = { ...solutionToRedesignWhip, accepted: true };
+    jest.spyOn(prisma.proposed_Solution, 'update').mockResolvedValueOnce(updatedSolution);
+    jest.spyOn(prisma.wBS_Element, 'findUnique').mockResolvedValueOnce(redesignWhipWBSElement);
+
+    const myProject = {... someProject.project, googleDriveFolderLink: 'https://drive.google.com/drive/folders/1', slideDeckLink: 'https://docs.google.com/presentation/d/1', bomLink: 'https://docs.google.com/spreadsheets/d/1', taskListLink: 'https://docs.google.com/spreadsheets/d/1', teamId: "1"};
+
+    jest.spyOn(prisma.project, 'findUnique').mockResolvedValueOnce(myProject);
+
+    const updatedProject = { ...someProject, budget: myProject.budget + solutionToRedesignWhip.budgetImpact};
+    const changes = [
+      {
+        changeRequestId: redesignWhip.crId,
+        implementerId: superman.userId,
+        detail: buildChangeDetail('Budget', String(myProject.budget), String(solutionToRedesignWhip.budgetImpact))
+      },
+      {
+        changeRequestId: redesignWhip.crId,
+        implementerId: superman.userId,
+        detail: buildChangeDetail('Duration', String(whipWorkPackage.duration), String(solutionToRedesignWhip.timelineImpact))
+      }
+    ];
+    const updatedWBSElement = {...redesignWhipWBSElement, changes: [changes]};
+    const updatedWorkPackage = { ...whipWorkPackage, project: updatedProject, duration: whipWorkPackage.duration + solutionToRedesignWhip.timelineImpact, wbsElement: updatedWBSElement }; 
+    jest.spyOn(prisma.work_Package, 'update').mockResolvedValueOnce(updatedWorkPackage);
```

Comment:
im not sure you need the whole updated object at all since it never gets used. i think you could just have it return null and it would have no effect on the functionality and would greatly simplifiy the tests. probably can do this in some other areas too